playground/frontend/playground_components/lib/widgets/split_view.dart

```diff
@@ -0,0 +1,261 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:provider/provider.dart';
+
+import '../constants/assets.dart';
+import '../constants/sizes.dart';
+
+enum SplitViewDirection {
+  vertical,
+  horizontal,
+}
+
+const _minRatio = 0.3;
+const _maxRatio = 0.7;
+const _defaultRatio = 0.5;
+const _dividerSize = BeamSizes.size8;
+
+class SplitView extends StatefulWidget {
```

Comment:
There should be `SplitViewController extends ValueNotifier<List<double>>` with an `n`-item list for `n` panes, adding to `1.0` (with a tiny tolerance for float-point error).
The widget should accept the controller and be stateless.
Min and max should be configurable per pane.

Reply:
Hi, I’ve committed the SplitViewController, but without List<double>